oecophylla/taxonomy/taxonomy.rule

```diff
@@ -200,19 +201,124 @@ rule taxonomy_kraken_combine_profiles:
         "benchmarks/taxonomy/taxonomy_kraken_combine_profiles.txt"
     run:
         pandas2biom(output[0], combine_profiles(zip(samples, input)))
-        if params['levels']:
-            for level in params['levels'].split(','):
-                redists = ['%s.redist.%s.txt' % (x[:-12], level)
-                           for x in input]
-                pandas2biom('%s_redist.%s.biom' % (output[0][:-13], level),
-                            combine_bracken(zip(samples, redists)))
+        for level in params['levels'].split(','):
+            redists = ['%s.redist.%s.txt' % (x[:-12], level)
+                       for x in input]
+            pandas2biom('%s_redist.%s.biom' % (output[0][:-13], level),
```

Comment:
I'm not too crazy about these sorts of magic numbers.  What is column `13`?

Reply:
agreed. Possible to parse table headers and address with labels?

Reply:
This `[:-13]` is to remove `_profile.biom` from `xxx_profile.biom`. Is there a better way for doing so?